libbeat/beat/event.go

```diff
@@ -51,7 +51,7 @@ func (e *Event) SetID(id string) {
 	if e.Meta == nil {
 		e.Meta = common.MapStr{}
 	}
-	e.Meta["id"] = id
```

Comment:
Given the special nature of this field name and the desire to keep it consistent in multiple places, do you think we should make it an exported const?

Reply:
We have more than one field that is special to meta. Let's clean these up (the other fields as well) in a follow up PR.